manifest_staging/charts/eraser/values.yaml

```
@@ -43,3 +43,4 @@ scanner:
     args: []
 
 nameOverride: ""
+eraserPullSecrets: []
```

nit: any reason we want to use eraser?
```suggestion
imagePullSecrets: []
```

---

done